src/shared/components/keep/StoreList.tsx

```
@@ -0,0 +1,147 @@
+                  transition={{ duration: 0.2 }}
+                  className={`absolute right-0 top-[130%] rounded-2xl bg-white px-3 py-2 shadow-lg`}
+                >
+                  <ul className="whitespace-nowrap text-center text-sm font-medium">
+                    {(Object.keys(RESTAURANT_ORDER_BY) as RestaurantOrderKey[]).map((ord) => (
+                      <li
+                        className={`cursor-pointer border-b px-2 py-1 ${orderBy === ord ? 'text-[var(--primary)]' : 'text-[var(--gray-5)]'}`}
+                        onClick={() => {
+                          setOrderBy(ord);
+                          setIsOrderDrop(false);
+                        }}
+                      >
```

Biome (2.4.6)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)